### PR TITLE
Rework sections on joint photometric fitting.

### PR DESCRIPTION
**Pull request by TallJimbo**
Split joint photometric fitting into two components, tract-level and global, and explain that we're not certain we'll actually do the global fitting at all.

Most of the content here is still Peter Yoachim's, but as I essentially split up the section he wrote in two, I've touched almost every sentence (but frequently only superficially).

### CODE REVIEW

File: sections/algorithmic_components.tex [hunk 1]
```diff
@@ -544,36 +544,43 @@ \subsubsection{Single Visit}
 \end{itemize}
 
 
-
 \subsubsection{Joint Multi-Visit}
 \label{sec:acJointPhotometricFit}
 
-For DRP, all the observations can be combined in the so-called ubercal procedure to generate the best possible measurement of the relative flux of each source.
+In joint photometric calibration, all observations within a tract are combined to jointly estimate the best possible measurement of the relative flux of each source and a (gray) photometric scaling for each visit.
```
Comment (RobertLuptonTheGood) [inline sections/algorithmic_components.tex]: Where the scaling's a function of position

File: sections/algorithmic_components.tex [hunk 2]
```diff
@@ -544,36 +544,43 @@ \subsubsection{Single Visit}
...
+
+\subsubsection{Global Fitting}
+\label{sec:acGlobalPhotometricFit}
+
+Global photometric fitting (as opposed to tract-level \hyperref[sec:acJointPhotometricFit]{Joint Multi-Visit Photometric Fitting}) will most likely not be run in Data Release Production, as we expect to be able to use the Gaia catalogs to calibrate between tracts.  The suitability of the Gaia catalogs for this purpose still needs to be confirmed in detail (it depends on our ability to type stars using the a combination of Gaia BP/RP spectra and LSST colors).  Even if Gais is used, a global photometric fit may still be useful as a QA tool (run after DRP) to verify the quality of the global photometric.
```
Comment (RobertLuptonTheGood) [inline sections/algorithmic_components.tex]: Not to much type them as predict all the LSST colours -- which may require restricting ourselves to a range of colours which is (I agree) tantamount to typing.  I'm hoping to be able to do this from only the RP/BP spectra, but don't yet know if this is possible.

Also typo "Gais" for Gaia.  "Psudo" for Pseudo

File: sections/algorithmic_components.tex [hunk 3]
```diff
@@ -544,36 +544,43 @@ \subsubsection{Single Visit}
...
-For DRP, all the observations can be combined in the so-called ubercal procedure to generate the best possible measurement of the relative flux of each source.
+In joint photometric calibration, all observations within a tract are combined to jointly estimate the best possible measurement of the relative flux of each source and a (gray) photometric scaling for each visit.
 
+This includes:
 \begin{itemize}
-\item Derive SEDs for calibration stars from colors and reference catalog classifications.
-\item Utilize additional information from wavelength dependent photometric calibration built by calibration products production to convert observed flux to a flux through a standard atmosphere.
-\item Fit zeropoint and possibly perturbations to all CCDs on multiple visits simultaneously after matching to reference catalog.
+\item Deriving SEDs for calibration stars from colors and reference catalog classifications.
+\item Utilizing additional information from wavelength-dependent photometric calibration built by calibration products production to convert observed flux to a flux through a standard atmosphere.
```
Comment (RobertLuptonTheGood) [inline sections/algorithmic_components.tex]: I don't think that we know that the monitor telescope will provide everything we need.  E.g. is the y-band extinction through cloud always gray (no OH absorption) -- if not, we need to use stellar colours.

It is possible that we can separate the LSST data into "photometric" and "other" nights, and use only the photometric nights as inputs into übercal,  but I don't think I know this for sure.  As such, the \"ubercal code needs to be prepared to solve for some atmospheric effects (as does e.g. the DES calibration code).

File: sections/algorithmic_components.tex [hunk 4]
```diff
@@ -544,36 +544,43 @@ \subsubsection{Single Visit}
...
...be tested on precursor datasets, such as images taken from the HSC and DECam instruments, and these lack the detailed characterization of wavelength-dependent photometric transmission effects planned for LSST.  Even when LSST commissioning data is available, the LSST atmospheric monitoring telescope, the Gaia catalog, and monochromatic flats may not be.  In order to exercise and commission the pipelines before the full system is operational we will have to include fitting for these chromatic effects in a global photometric fit that can be run on important precursor datasets.  This will not obtain the same level of accuracy as the full LSST system, but it will provide wavelength- and spatially-dependent photometric calibrations that are sufficiently accurate to test our ability to apply those corrections to our measurements.  When the full LSST photometric calibration system is operational, we expect this functionality to continue to play a role in QA even though it will probably not be run as part of Data Release Production.  Obviously, having this capability also serves as risk mitigation in case some aspects of the planned photometric calibration system fail to perfom adequately.
```
Comment (RobertLuptonTheGood) [inline sections/algorithmic_components.tex]: Calypso should be available before comCam

I don't think I know that using stars to constrain e.g. the water vapour in the atmosphere will be inferior to the calypso data.

Comment (TallJimbo) [reply sections/algorithmic_components.tex]: > I don't think I know that using stars to constrain e.g. the water vapour in the atmosphere will be inferior to the calypso data.

But we are treating using Calypso as the baseline, with the big global photometric fit the backup plan, right?  I agree scientifically that it's unclear which will be better (or I'm willing to buy it; I don't have an opinion), but I think what I've written is consistent with the current plan.

File: sections/algorithmic_components.tex (hunk 2 above)
Comment (TallJimbo) [reply sections/algorithmic_components.tex]: Fixed.

File: sections/algorithmic_components.tex (hunk 1 above)
Comment (TallJimbo) [reply sections/algorithmic_components.tex]: Clarified.

File: sections/algorithmic_components.tex (hunk 3 above)
Comment (TallJimbo) [reply sections/algorithmic_components.tex]: I've added this caveat.

**Pull request merged**